tf_yarn/_dispatch_task.py

```diff
@@ -217,6 +223,7 @@ def _setup_logging(log_conf_file):
     parser.add_argument("--num-ps", type=int)
     parser.add_argument("--experiment-fn", type=load_fn)
     parser.add_argument("--log-conf-file", type=str)
+    parser.add_argument("--num-threads",type=int)
     args = parser.parse_args()
     _setup_logging(args.log_conf_file)
-    main(args.experiment_fn, list(iter_tasks(args.num_workers, args.num_ps)))
```

Comment:
as comment above. rename to override_threads_parallelism_config (ore sth similar)

Reply:
yes, have added num_threads in as input arg--- main(args.experiment_fn, list(iter_tasks(args.num_workers, args.num_ps)),args.num_threads) .